CONTRIBUTING.rst

```diff
@@ -862,6 +865,19 @@ commands:
     # Check JS code in .js and .html files, report any errors/warnings and fix them if possible
     yarn run lint:fix
 
+How to sync your fork
+=====================
+
+When you have your fork, you should periodically synchronize the master of your fork with the
+Apache Airflow master. In order to do that you can ``git pull`` to your local git repository from
```

Comment:
```suggestion
Apache Airflow master. In order to do that you can ``git pull --rebase`` to your local git repository from
```
Should we explicitly recommend fetch and rebase? Just in case someone one read this and skips rest of the information